css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 225px;
+  bottom: 225px;
```

Comment:
Oh no, this is a _**big**_ movement. In general, you should try to use the box offset properties closest to the edge you want to position this element at. In this case, that is `top` and `right`